filters/builtin/comment.go

```diff
@@ -0,0 +1,23 @@
+package builtin
+
+import "github.com/zalando/skipper/filters"
+
+type comment struct{}
+
+// NewComment is a filter to comment a filter chain. It does nothing
```

Comment:
We'd need an entry in filters.md docs.

Another/additional idea is to introduce `annotation(<key>, <value>)` filter that represents metadata and could be used by route pre-processing for something. The subject usecase could then become `annotation("comment", "foo bar baz")`.

For this filter we may also add a pre-processor that will skip this filter and therefore there will be no penalty in runtime (e.g. makes no sense to update filter metrics for it).

Reply:
Also we should start a countdown until someone puts json into the value :)

Reply:
I think it overcomplicates the problem. Given that the implementation at runtime will be literally:

```
TEXT local.github.com/zalando/skipper/filters/builtin.(*comment).Request(SB) /home/sszuecs/go/src/github.com/zalando/skipper/filters/builtin/comment.go
  comment.go:20         0x15f12c0               c3                      RET
```

Reply:
> For this filter we may also add a pre-processor that will skip this filter

It should be post-processor I think, otherwise routesrv will not show this filter in the routing table (need to check)

Reply:
implemented, but I think it's not worth it